File: src_py/week0/number_swap.py
class NumberSwap:
    def __init__(self):
        self.name = "number_swap"
        self.a = 0
        self.b = 0

    def get_name(self):
        return self.name

    def process(self):
        self.a = input("Input first integer: ")
        self.b = input("Input second integer: ")
        if self.a > self.b:
            temp = self.a
            self.a = self.b
            self.b = temp
        print(self.a, self.b)

File: src/com/week0/NumberSwap.java
package com.week0;

import com.Option;

import java.util.Scanner;

public class NumberSwap extends Option {

    public NumberSwap() {
        optionName = "Number Swap";
    }

    public String getOptionName() {
        return optionName;
    }

    private void swap() {
        Scanner scanObj = new Scanner(System.in);

        System.out.println("Input first integer:");
        int a = scanObj.nextInt();
        System.out.println("Input second integer:");
        int b = scanObj.nextInt();

        if (a > b) {
            int temp = a;
            a = b;
            b = temp;
        }
        System.out.println(a + " " + b);
    }


    public void process(Scanner scanObj) {
        // super.process() will just run the process() method from Option.java
        super.process(scanObj);
        // Then run swap()
        NumberSwap testObj = new NumberSwap();
        testObj.swap();
    }
}
